doc/_extensions/software_maturity_table.py

```diff
@@ -223,9 +251,15 @@ def run(self):
 
             for soc in all_socs:
                 entry = nodes.entry()
-                if soc in maturity_list["Experimental"]:
+                footnote_target = None
+
+                if soc in experimental:
+                    if self.minimum_version(version, "0.2.0") and experimental[soc]:
```

Comment:
As we are auto-detecting the format of `boards_and_shields` as `list` or `dict` this is already backwards compatible, why do we need this version check here? The version of the script is anyway bumped to `0.2.0`, so, won't this always be true?